scripts/add-helm-code.sh

```diff
@@ -21,13 +21,26 @@
 # Working directory does not matter.
 
 HOME_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )"
+# Directory containing the yaml file
+dir=${HOME_DIR}/chart/templates
 
-if [ "$#" -lt 1 ]; then
-    echo "adds/remove helm tags to cluster scoped files such as ClusterRole and CLusterRoleBinding"
-    echo "Usage: $(basename $0) add | remove"
-    exit
-fi
-CMD=$1
+while (( $# > 0 )); do
+	case "$1" in
+    (-h|--help)
+    	echo "adds/remove helm tags to cluster scoped files such as ClusterRole and CLusterRoleBinding"
+	    echo "Usage: $(basename $0) [--dir directory-containing-yaml] add | remove"
```

Comment:
I do not think we know that `$(basename $0)` transforms `$0` into something that works with the user's current `$PATH`.

Reply:
This is was the pre-existing script in this file.  I humbly suggest it's outside of the scope of these changes.

Reply:
You're right, this PR did not introduce this problem.

Reply:
So this does not have to be fixed in this PR.